### ISSUE TITLE
vivisect.exc.DuplicateName exception

### ISSUE THREAD

**Issue by ryanlovett** [mandiant/capa#1464]
### Description

I ran capa for the first time on some known malware and it produced an exception.

```bash
% ./capa fake-systemd
Traceback (most recent call last):
  File "main.py", line 1325, in <module>
  File "main.py", line 1228, in main
  File "main.py", line 560, in get_extractor
  File "main.py", line 480, in get_workspace
  File "viv_utils/__init__.py", line 117, in getWorkspace
  File "vivisect/__init__.py", line 2786, in loadFromFile
  File "vivisect/parsers/elf.py", line 31, in parseFile
  File "vivisect/parsers/elf.py", line 371, in loadElfIntoWorkspace
  File "vivisect/parsers/elf.py", line 115, in makeFunctionTable
  File "vivisect/__init__.py", line 2623, in makeName
vivisect.exc.DuplicateName: Duplicate Name: systemd.fini_function_0 at 0x00a6122c and 0x00438a60
[1433338] Failed to execute script 'main' due to unhandled exception!
```
### Steps to Reproduce

1. Download and extrace capa-v5.1.0-linux.zip
2. Run `capa` against some known malware.

**Expected behavior:**

I wasn't certain what to expect since I've never run `capa` before, although I generally expected it to run without error.

**Actual behavior:**
It raised an exception.

### Versions

capa 5.1.0
Ubuntu 20.04
I tried with python 3.11 from mambaforge and python 3.8.10 from the distribution.

I also ran on macOS 13.3.1 with the same executable and same results.

**Comment by mr-tz** [comment]
Thanks for reporting this! Can you share the sample (hash) by any chance?

**Comment by ryanlovett** [comment]
Do you mean share the file? Yes -- what is the process for doing so? It is known malware so I'm hesitant to share it publicly.

**Comment by mr-tz** [comment]
If it's on VirusTotal etc. the hash is fine. Otherwise you can upload or if you prefer send it via email to me.

**Comment by williballenthin** [comment]
making a guess without seeing the file: if there are multiple `.fini_array` sections in the ELF file, then vivisect may generate colliding names here: https://github.com/vivisect/vivisect/blob/1c1f5586349b058115db03ec161224609ebc1044/vivisect/parsers/elf.py#L441

this is ultimately a vivisect bug, but we can triage here and propose a fix to help out.

**Comment by ryanlovett** [comment]
Thanks, I found it on VirusTotal: `b5099a3b0ddce485a0d1e31cde4fc84d3ecd26464a75f223b185569ea734fafb`

**Comment by mr-tz** [comment]
Thanks! I can reproduce the error and will investigate further.

**Comment by mr-tz** [comment]
This is fixed in vivisect `v1.1.1`, so we just need a new capa release.

**Comment by mr-tz** [comment]
Fixed in latest releases.
